gatsby-config.js

```diff
@@ -143,6 +142,66 @@ module.exports = {
               escapeEntities: {},
             },
           },
+          {
+            resolve: `gatsby-plugin-feed`,
+            options: {
+              query: `
+                {
+                  site {
+                    siteMetadata {
+                      title
+                      description
+                      siteUrl
+                      site_url: siteUrl
+                    }
+                  }
+                }
+              `,
+              feeds: [
+                {
+                  serialize: ({ query: { site, allMarkdownRemark } }) => {
+                    return allMarkdownRemark.edges.map(edge => {
+                      return Object.assign({}, edge.node.frontmatter, {
+                        description: edge.node.excerpt,
+                        date: edge.node.frontmatter.date,
+                        url: site.siteMetadata.siteUrl + edge.node.fields.slug,
+                        guid: site.siteMetadata.siteUrl + edge.node.fields.slug,
+                        custom_elements: [{ "content:encoded": edge.node.html }],
+                      })
+                    })
+                  },
+                  query: `
+                    {
+                      allMarkdownRemark(
+                        sort: { order: DESC, fields: [frontmatter___date] },
+                        filter: {fileAbsolutePath: {regex: "content/blog/"}},
+                        limit: 10
+                      ) {
+                        edges {
+                          node {
+                            excerpt
+                            html
+                            fields { slug }
+                            frontmatter {
+                              title
+                              date
+                            }
+                          }
+                        }
+                      }
+                    }
+                  `,
+                  output: "/rss.xml",
+                  title: "Your Site's RSS Feed",
```

Comment:
Could we set this to something a bit better?
```suggestion
                  title: "Ardalis RSS Feed",
```